PickAPic/PickAPic/ViewController.m

```diff
@@ -2,17 +2,85 @@
 //  ViewController.m
 //  PickAPic
 //
-//  Created by Michael Kavouras on 9/13/15.
-//  Copyright © 2015 Mike Kavouras. All rights reserved.
+//  Created by Varindra Hart on 9/17/15.
+//  Copyright © 2015 Varindra Hart. All rights reserved.
 //
 
 #import "ViewController.h"
+#import <QuartzCore/QuartzCore.h>
 
-@interface ViewController ()
+@interface ViewController ()<UIImagePickerControllerDelegate, UINavigationControllerDelegate, UIAlertViewDelegate>
+
+@property (nonatomic) IBOutlet UIImageView *imageView;
+@property (weak, nonatomic) IBOutlet UIButton *takeAPicButton;
+@property (weak, nonatomic) IBOutlet UIButton *useCameraRollButton;
 
 @end
 
 @implementation ViewController
 
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.takeAPicButton.layer.cornerRadius = 10;
+    self.useCameraRollButton.layer.cornerRadius = 10;
+    self.navigationItem.title = @"Pick A Pic";
+    
+    if (![UIImagePickerController isSourceTypeAvailable:UIImagePickerControllerSourceTypeCamera]) {
+        
+//        UIAlertController * alert = [UIAlertController alertControllerWithTitle:@"Warning" message:@"Attempting to run application on a non camera compliant device" preferredStyle:UIAlertControllerStyleAlert];
+//        [alert addAction:[UIAlertAction actionWithTitle:@"Cancel" style:UIAlertActionStyleCancel handler:nil]];
+//        [alert show];
+
+        UIAlertView * alert = [[UIAlertView alloc] initWithTitle:@"Warning" message:@"Attempting to run application on a non camera compliant device" delegate:self cancelButtonTitle:@"Cancel" otherButtonTitles:nil];
+        [alert show];
+    }
+    // Do any additional setup after loading the view, typically from a nib.
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+
+- (IBAction)takePic:(UIButton *)sender{
+    
+    UIImagePickerController *picker = [[UIImagePickerController alloc]init];
+    picker.delegate = self;
+    picker.allowsEditing = YES;
+    picker.sourceType = UIImagePickerControllerSourceTypeCamera;
```

Comment:
Should check to see that the source type exists right before you set it as opposed to in viewDidLoad